extensions/amp-story/1.0/amp-story-system-layer.js

```diff
@@ -340,10 +389,38 @@ export class SystemLayer {
    * @private
    */
   onMutedStateUpdate_(isMuted) {
+
     this.vsync_.mutate(() => {
-      isMuted ?
-        this.getShadowRoot().setAttribute(AUDIO_MUTED_ATTRIBUTE, '') :
-        this.getShadowRoot().removeAttribute(AUDIO_MUTED_ATTRIBUTE);
+      if (isMuted) {
+        this.getShadowRoot().setAttribute(AUDIO_MUTED_ATTRIBUTE, 'muted');
+      } else if (this.storeService_.get(StateProperty.PAGE_HAS_AUDIO_STATE)) {
+        this.getShadowRoot().setAttribute(AUDIO_MUTED_ATTRIBUTE, 'audioon');
+      } else {
+        this.getShadowRoot().setAttribute(AUDIO_MUTED_ATTRIBUTE, 'noaudio');
+      }
+
+    });
+  }
+
+  /**
+   * Hides element after elapsed time
+   * @param {int} hideTimeout
+   * @private
+   */
+  hideAfterTimeout_(hideTimeout) {
```

Comment:
Because the `hideTimeout` is always the same, should we simply use a constant at the top of this file?